save-orchestrator/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,14 @@
+spring.h2.console.enabled=true
```

Comment:
Is it by mistake? We don't need DB setttings in orchestrator

Reply:
Fixed